receiver/cudareceiver/README.md

```diff
@@ -0,0 +1,26 @@
+# CUDA (GPU) Receiver
+
+**Status: alpha**
+
+## Prerequisites
+
+CUDA Receiver is currently only supported on the following environment.
+
+* OS: Ubuntu 18.04
+* CPU architecture: x86_64
+* CUDA Toolkit: version 10.2
+
+Technically, it depends upon `nvml.h` and `libnvidia-ml.so`, and it should work on the environment where CUDA Toolkit version 10.2 is available.
```

Comment:
I meant you should document here what's required at compile time vs build time, "depends on" isn't clear. So it sounds like both header and so are required at compile time and so only at runtime? Can you add that here?

Reply:
SG. I made it clear that the header is required on build time and dynamic lib is required both on build and run time.